PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4170,6 +4170,8 @@ LRESULT FAR PASCAL FindReplaceDlg::comboEditProc(HWND hwnd, UINT message, WPARAM
 
 	bool isDropped = ::SendMessage(hwndCombo, CB_GETDROPPEDSTATE, 0, 0) != 0;
 
+	static TCHAR draftString[FINDREPLACE_MAXLENGTH] = { '\0' };
+

```

Comment:
Since more than just find and replace comboboxes are subclassed with this proc, I'm not sure using FINDREPLACE_MAXLENGTH is the best choice here.  Do I know what would be better?  Nope. :-)
I suppose comparing hwndCombo to see if it is either the find box or the replace box would make it reasonable to then use FINDREPLACE_MAXLENGTH.

Reply:
Yeah, I took that from other parts of this code file like L50, L479, L5401, etc.

Reply:
But looking at the global sizes available, I think this is pretty much the only choice: 
![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/16470072/bc9a52a4-032b-4588-ab7a-145ad6310902)